### PR TITLE
API update

### PR DESCRIPTION
**Pull request by therfert**
##### SUMMARY
- support API paths ``ip arp``, ``ip firewall raw``, ``ipv6 firewall raw``
- defaults corrected for fields in ``interface wireguard peers`` API path

##### ISSUE TYPE
- Bugfix Pull Request
- Feature Pull Request

##### COMPONENT NAME
api_modify, api_info

### CODE REVIEW

File: plugins/module_utils/_api_data.py [hunk 1]
```diff
@@ -1184,14 +1184,14 @@ def join_path(path):
...
-            'endpoint-address': KeyInfo(),
+            'endpoint-address': KeyInfo(default=''),
             'endpoint-port': KeyInfo(default=0),
             'interface': KeyInfo(),
-            'persistent-keepalive': KeyInfo(),
-            'preshared-key': KeyInfo(),
+            'persistent-keepalive': KeyInfo(can_disable=True, remove_value=0),
+            'preshared-key': KeyInfo(can_disable=True, remove_value=''),
             'public-key': KeyInfo(),
         },
```
Comment (felixfontein) [inline plugins/module_utils/_api_data.py:1196]: CC @cosandr

Comment (cosandr) [reply plugins/module_utils/_api_data.py:1196]: LGTM, causes no changes for my config and `allowed-address` is indeed required.

**Review by felixfontein — APPROVED** [review]
Also looks good to me!

**Comment by felixfontein** [comment]
@therfert thanks for your contribution!
@cosandr thanks for reviewing!

**Pull request merged**